app/views/documents/fields/_govspeak_input.html.erb

```diff
@@ -1,5 +1,5 @@
 <div class="govuk-grid-row">
-  <div class="govuk-grid-column-two-thirds">
+  <div class="govuk-grid-column-two-thirds" id="body">
```

Comment:
It looks like this conflicts with the id used for the label, see the test fails here: https://ci.integration.publishing.service.gov.uk/job/content-publisher/job/JS-prompt-targets/2/console

It's worth checking whether other ids you have created conflict with existing ones.